telemetry/definitions/commonDefinitions.json

```diff
@@ -336,6 +336,11 @@
             "allowedValues": ["primary", "localSecondary", "globalSecondary"],
             "description": "The type of index being hit for the query/scan operation"
         },
+        {
+            "name": "dynamoDbTarget",
```

Comment:
CWL uses `cloudWatchResourceType` for a similar concept but in this case we seem to be combining a resource type with abstractions associated with it? I don't think we need to separate the two ideas necessarily as the idea of a resource is abstract anyway. But at least for the DBB service spec a table is clearly a resource while the properties of it are not. I'm just thinking here, no changes needed.